bucket/stevedore.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Overall, I am a bit worried that this script duplicates lots of Stevedore installer logic. And things are guaranteed to diverge with later changes to Stevedore. Is it a common Scoop approach or it is possible to reuse more bits from upstream MSI?

Reply:
Actually, I realized I can use `msiexec` and use `TARGETDIR` to specify a directory, which is sufficient for nonportable. Would it be better to run with a dialog box/progress bar or completely unattended?